Gordon360/Services/HousingService.cs

```diff
@@ -34,36 +34,73 @@ public IEnumerable<HousingViewModel> GetAll()
 
```

Comment:
GetAll is unnecessary, correct? We should make sure it is and delete it if so

Reply:
I think It was from the end-to-end hello world and is no longer needed